clicommand/commands.go

```diff
@@ -82,6 +82,7 @@ var BuildkiteAgentCommands = []cli.Command{
 		Usage: "Utility commands, intended for users and operators of the agent to run directly on their machines, and not as part of a Buildkite job",
 		Subcommands: []cli.Command{
 			KeygenCommand,
+			ToolSignCommand,
```

Comment:
should we either rename `KeyGen` to `ToolKeygen` or rename `ToolSign` to `Sign`?

Reply:
I think you know which one I will choose 😂